.circleci/config.yml

```diff
@@ -3,6 +3,16 @@ defaults: &defaults
   docker:
     - image: ubuntu:14.04
 jobs:
+  sanity_check:
+    <<: *defaults
+    steps:
+      - checkout
+      - run:
+          name: Setup requirements
+          command: apt-get update && apt-get -y install git astyle sed
```

Comment:
Please use setup_common_requirements.sh for that

Reply:
Done.

Reply:
Note that this step doesn't need most of what's in `setup_common_requirements.sh`. Only git, sed and astyle are needed there.